force-app/main/default/classes/SOQL.cls

```diff
@@ -85,20 +94,30 @@ public with sharing class SOQL {
      * @return A map of Task Ids to Task records.
      * Update the corresponding test class and testGetTasksDueNextWeek method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
-
+    public static Map<Id, Task> getTasksDueNextWeek() {
+        Date today = Date.today();
+        Date nextWeek = today.addDays(7);
+        
+        Map<Id, Task> tasksMap = new Map<Id, Task>();
+        
+        for (Task t : [SELECT Id, Subject, ActivityDate FROM Task WHERE ActivityDate >= :today AND ActivityDate <= :nextWeek]) {
```

Comment:
Great work! You can also use the date literal NEXT_N_DAYS:n to find tasks due in the next 7 days